# remove hardcoded paths from scripts && set MKL to false (using FOSS toolchain)
# author: Denis Kristak + Kenneth Hoste (HPC-UGent)
--- r-inla-Version_21.05.02/rinla/inst/bin/linux/64bit/inla.run.orig	2021-05-02 21:30:04.000000000 +0200
+++ r-inla-Version_21.05.02/rinla/inst/bin/linux/64bit/inla.run	2021-05-18 15:17:11.175865393 +0200
@@ -13,15 +13,15 @@
     fi
 fi
 
-for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
-    if [ -d "$d" ]; then
-	if [ -z "$D" ]; then
-	    D="$d"
-	else
-	    D="$D:$d"
-	fi
-    fi
-done
+# for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
+#     if [ -d "$d" ]; then
+# 	if [ -z "$D" ]; then
+# 	    D="$d"
+# 	else
+# 	    D="$D:$d"
+# 	fi
+#     fi
+# done
 
 for f in $DIR/first/lib*.so*; do
     case "$f" in
--- r-inla-Version_21.05.02/rinla/R/options.R.orig	2021-05-18 15:17:11.175865393 +0200
+++ r-inla-Version_21.05.02/rinla/R/options.R	2021-05-18 15:27:35.211947369 +0200
@@ -101,7 +101,7 @@
             num.threads = paste0(parallel::detectCores(all.tests = TRUE, logical = FALSE), ":1"),
             blas.num.threads = 0L,
             smtp = "default",
-            mkl = if (inla.os("linux") || inla.os("mac")) TRUE else FALSE,
+            mkl = FALSE,
             vecLib = if (inla.os("mac")) TRUE else FALSE,
             vecLibPath = "", 
             pardiso.license = NULL,
--- r-inla-Version_21.05.02/rinla/inst/bin/linux/64bit/fmesher.run.orig	2021-05-25 11:45:08.870229000 +0200
+++ r-inla-Version_21.05.02/rinla/inst/bin/linux/64bit/fmesher.run	2021-05-25 11:51:42.965833000 +0200
@@ -4,17 +4,17 @@
 prog=${tmp%%.run}
 DIR=$(dirname "$cmd")
 
-D=""
-for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
-    if [ -d "$d" ]; then
-	if [ -z "$D" ]; then
-	    D="$d"
-	else
-	    D="$D:$d"
-	fi
-    fi
-done
-export LD_LIBRARY_PATH="$DIR/first:$D:$DIR:$LD_LIBRARY_PATH"
+#D=""
+#for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
+#    if [ -d "$d" ]; then
+#	if [ -z "$D" ]; then
+#	    D="$d"
+#	else
+#	    D="$D:$d"
+#	fi
+#    fi
+#done
+#export LD_LIBRARY_PATH="$DIR/first:$D:$DIR:$LD_LIBRARY_PATH"
 
 if [ "${INLA_DEBUG}XX" != "XX" ]; then
     echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
